### PR TITLE
Do some of the todos

### PR DESCRIPTION
**Pull request by benthecarman**

### CODE REVIEW

File: harbor-ui/src/main.rs [hunk 1]
```diff
@@ -737,7 +742,6 @@ impl HarborWallet {
                         _ => "Unknown".to_string(),
                     };
 
-                    // TODO: what to do about balance in this case? Maybe it should be Option<u64>?
```
Comment (benthecarman) [inline harbor-ui/src/main.rs:740]: we didnt need the balance here

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/target
```
Comment (futurepaul) [inline .gitignore:1]: on my system running `just run` from the root dir still creates a target dir in the root. so maybe we need `/target` and `*/target`?

File: harbor-ui/src/main.rs [hunk 3]
```diff
@@ -448,8 +448,14 @@ impl HarborWallet {
                         let amount = if self.is_max {
                             None
                         } else {
-                            // TODO: error handling
-                            Some(self.send_amount_input_str.parse::<u64>().unwrap())
+                            match self.send_amount_input_str.parse::<u64>() {
+                                Ok(amount) => Some(amount),
+                                Err(e) => {
+                                    eprintln!("Error parsing amount: {e}");
```
Comment (futurepaul) [inline harbor-ui/src/main.rs:454]: nice I can show a toast for these parse fails. I also need to fix the amount input so you can only type numbers

**Review by futurepaul — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (benthecarman) [reply .gitignore:1]: thanks, fixed

File: harbor-ui/src/main.rs (hunk 3 above)
Comment (benthecarman) [reply harbor-ui/src/main.rs:454]: > I also need to fix the amount input so you can only type numbers

i tried to do this but couldnt figure it out

**Review by futurepaul — APPROVED** [review]

**Pull request merged**